a3p-integration/proposals/z:acceptance/wallet.test.js

```diff
@@ -44,7 +44,7 @@ const getBalance = async (addresses, targetDenom = undefined) => {
   return addresses.length === 1 ? balancesList[0] : balancesList;
 };
 
-test.serial(`send invitation via namesByAddress`, async t => {
+test.only(`send invitation via namesByAddress`, async t => {
```

Comment:
did you mean to push this?

Reply:
No, I am working on the requested changes and by mistake end up pushing a commit with work in progress.
I already reverted that commit.

Sorry for the confusion.